packs/equipment/serpent-dagger.json

```diff
@@ -51,7 +51,17 @@
         "reload": {
             "value": "-"
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "Note",
+                "title": "{item|name}",
+                "text": "When you critically succeed at an attack roll with the serpent dagger, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.fesd1n5eVhpCSS18]{Sickened 1} unless it succeeds at a @Check[type:fortitude|dc:19|traits:poison] save. This is a poison effect.",
```

Comment:
Additionally, it's not wrong per say, but you don't need to specify the poison trait in your inline save. It will be inherited from the serpent dagger automatically.

Reply:
Okay, what do you mean by extract packs? My current method which I assume is incorrect is to just right click and export item and copy over the changes to the rule elements.

Also I'll update this to be correct later tonight

Reply:
Check out the wiki page: https://github.com/foundryvtt/pf2e/wiki/Helping-with-Data-Entry

Reply:
https://github.com/foundryvtt/pf2e/wiki/Helping-with-Data-Entry#getting-the-changes-out-of-foundry